### PR TITLE
 description added to diagram

### PR DESCRIPTION
**Pull request by urviljoshi**

### CODE REVIEW

File: docs/websub.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ In MOSIP WebSub is used to share data with services and partners.  [Kafka](https
 
 ![](_images/websub.png)
 
+The relationship of Websub with other services is explained here.NOTE: The numbers do not signify sequence of operations or control flow
```
Comment (SwatiPeriwal) [inline docs/websub.md]: Websub to be written as "WebSub". Please maintain consistency as per the terms used in docs.mosip.io

File: docs/websub.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ In MOSIP WebSub is used to share data with services and partners.  [Kafka](https
...
+
+1. Topic is registered and published or subscribed by Mosip Services.
+2. Content delivery, Intent Verification is done Websub to Mosip Services.
```
Comment (SwatiPeriwal) [inline docs/websub.md]: Sentence needs correction - Content delivery, Intent Verification is done Websub to Mosip Services.

File: docs/websub.md (hunk 1 above)
Comment (urviljoshi) [reply docs/websub.md]: done

File: docs/websub.md (hunk 2 above)
Comment (urviljoshi) [reply docs/websub.md]: done

**Review by SwatiPeriwal — APPROVED** [review]
Reviewed and merging the points.

**Comment by SwatiPeriwal** [comment]
Merged

**Pull request merged**